### ISSUE TITLE
[BUG]Ensure the Reorder List is accessible

### ISSUE THREAD

**Issue by alxzak** [kirbydesign/designsystem#4028]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
The reorderlist is not accessible/keyboard enabled yet.

## What was the expected behavior?
You should be able to:
- Navigate to the list using the Tab key.
- Tab to an individual item within the list.
- Press Command (macOS) or Control (Windows) to activate move mode, with assistive technology announcing that the item is now movable.
- Visually identify that the item is ready for relocation (e.g., through a visual cue).
- Use the Up and Down Arrow keys to reposition the item, while hearing auditory feedback indicating the new position.
- Release Command/Control to confirm the move and receive confirmation that the item has been relocated.

## Are there any additional context?
You can read more about it here: https://pressbooks.library.torontomu.ca/wafd/chapter/sortable-lists/
The visual style is currently being developed by other teams and should be approved before we begin implementation.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
